README.md

```diff
@@ -19,10 +19,32 @@
    This will start a webpack-dev-server instance at port 8080 and start a
    watcher that will rebuild the browser extension with every file change.
 
-1. In Firefox 57 + open `about:debugging` and load the
-   `build/extension/manifest.json` file.
+1. To activate the extension:
 
-1. Visit `http://localhost:8080` to see the web-based theme editor - changes
+   1. Use Firefox 57 + Dev or Nightly browser
+
+   2. Go to `about:config` and add setting `xpinstall.signatures.required` and set to `false`
+
+   3. Find the XPI for the environment:
+
+      - Locally: `npm run package` which adds an addon.xpi to the root of the project
+
+      - DEV / STAGE: Visit the dev or stage version of the website and click on "Get Firefox Color" (i.e. open testing.html) and use one of the referenced XPI files
+
+   4. Now load the XPI (from the previous step) to the browser by one of the following ways:
+
+      - Go to `about:debugging` and click on "Load Temporary Add-on..." and add the xpi
+
+      - (or) drag and drop the XPI to the browser
```

Comment:
Step 2 should be combined with this bullet point. When the "Load Temporary Add-on" button is used, the preference change is not needed.

And also note that this alternative only works on Beta, Developer Edition and Nightly.

Reply:
@Rob--W, just to make sure i have this right before I change, would be something like this:


1. Find the XPI for the environment:
    * Locally: `npm run package` which adds an `addon.xpi` to the root of the project 
    * DEV / STAGE: Visit the dev or stage version of the website and click on "Get Firefox Color" (i.e. open testing.html) and use one of the referenced XPI files 

2. Now load the XPI (from the previous step) to the browser by one of the following ways:
    * Go to `about:debugging` and click on "Load Temporary Add-on..." and add the xpi 
    * (or) Go to about:config and add setting `xpinstall.signatures.required` and set to `false`. Next drag and drop the XPI to the browser. Note:  you must use Firefox 57 + Beta, Dev or Nightly browser w/ this approach.

Reply:
Looks good. You can drop the "57+" qualifier from your comments, by the way. Older versions aren't relevant and not worth mentioning.